### ISSUE TITLE
The Neighborhood row's Gloss has no fetch: decide the bounded read, or amend the four-field promise

### ISSUE THREAD

**Issue by tim-nish** [tim-nish/kogaki#689]
## The gap

kogaki#686 disposition 3 rules four fields per Neighborhood row: Strand ID, the relation in plain words, **the Gloss**, and the claim with its level. Three are produced. **The Gloss is not.**

`neighborhoodOf` emits no `gloss` on a suggestion; `cmdReport` sets `sug.gloss = sug.gloss || null` (`terrain/terrain.mjs:2703`); every production pull renders **`gloss unrecorded`**, and the regenerated golden specimen shows it.

That rendering is deliberate and is the *disclosure*, not the fix: an earlier revision substituted the slug, which put an unquoted identifier under a grammar declaring a served rendering — the paraphrase-standing-for-a-quote shape the kit's verbatim rule refuses.

## Why #686 could not decide it

A suggestion sits **outside the survey's candidates**, so its served headline needs a shard read the report path does not make. Gloss headlines are fetched only on the tag-scoped `view`/`cotags` path (`parseGlossShard`, `terrain/terrain.mjs:567`). Adding one to the report path is a **bounded-read decision** — §compose-input's one-shard-per-run discipline governs it — and #686 rules on the Neighborhood's shape, not on the seam's read budget.

## What is owed

1. Decide whether the report path may make a shard read for suggestion slugs, and under what bound.
2. If yes: fetch and join, and the row renders a served headline quoted at its cite.
3. If no: the `gloss unrecorded` disclosure stands and disposition 3's field list is amended to say the Gloss is unavailable by decision rather than by omission — the four-field row would otherwise keep promising a field nothing can fill.
4. Either way `report-format.json`'s `neighborhood_suggestion_gloss` class, which declares both arms today, is reconciled with the outcome.

## The implementation this splits from

`spec/686-neighborhood-v2` @ `072fc5f` implements #686's four dispositions and passes 18/18 checks. This issue is one of three decisions split out of #686 at its carrier-arbitration gate (2026-08-28) so the implementation can close and the decisions can be taken on their own carriers.

searched: terrain/terrain.mjs for gloss production on the neighborhood path (neighborhoodOf emits none; cmdReport:2703 defaults to null; parseGlossShard:567 and its callers are the tag-scoped view/cotags path only); specs/spec-terrain/SPEC.md §13.4 as amended by #686 (declares the four fields); specs/spec-terrain/report-format.json neighborhood_suggestion_gloss (declares both arms); the regenerated golden specimen at checks/fixtures/terrain/format/full-report.md (renders `gloss unrecorded`); kogaki#686 rounds on PR #687 and #688 (the finding raised four times across both, carried each time)

consult: none: the fork is a read-budget decision about this repository's own seam discipline, and the served surface was consulted at the sitting that raised it without discriminating a read bound

warrant: blocks: kogaki#686 disposition 3 promises a four-field row and ships three; until this is decided the grammar declares a field the pipeline cannot fill

remedy: detects: the disclosure arm makes the missing fetch visible on every pull rather than substituting the slug ; constrains: the row can no longer render an unquoted identifier in a field the grammar calls a served Gloss

searched: terrain/terrain.mjs for gloss production on the neighborhood path (neighborhoodOf emits none; cmdReport:2703 defaults to null; parseGlossShard:567 and its callers are the tag-scoped view/cotags path only); specs/spec-terrain/SPEC.md 13.4 as amended by #686; specs/spec-terrain/report-format.json neighborhood_suggestion_gloss; the regenerated golden specimen rendering 'gloss unrecorded'; kogaki#686 rounds on PR #687 and #688 where the finding was raised four times and carried each time

consult: none: the fork is a read-budget decision about this repository's own seam discipline, and the served surface was consulted at the sitting that raised it without discriminating a read bound

warrant: blocks: kogaki#686 disposition 3 promises a four-field row and ships three; until this is decided the grammar declares a field the pipeline cannot fill

remedy: detects: the disclosure arm makes the missing fetch visible on every pull rather than substituting the slug ; constrains: the row can no longer render an unquoted identifier in a field the grammar calls a served Gloss

**Comment by tim-nish** [comment]
## Two findings carried here from PR #692 round 2, landed on the carrier rather than only in a commit message

The round-2 report's own third finding is that the first of these was *"recorded in a commit message and nowhere on the carrier, which is how a round-2 finding with no third round goes missing."* Correct — this comment is the repair. #692 merged on a `done` verdict with these open.

### 1. `NeighborhoodLevel` is declared in a shape the reader cannot consume — and its note claims otherwise

`specs/spec-terrain/report-format.json:55` adds:

```json
"NeighborhoodLevel": "core|useful|background"
```

Every other token in that block is an **object carrying `shape`**. `tokenFragment` (`terrain/format-guard.mjs:52-56`) returns `FREE` unless `t.shape` exists **and** is anchored, so a bare string yields `FREE` and `neighborhood_suggestion_claim` compiles to:

```
^  [\s\S]*? \[[\s\S]*?\]\s*$
```

`  anything [anything]` is admitted. The class discriminates by its bracketed **structure** — which is why the coverage case passes and why it is not near-bare — but **not by the level vocabulary**, and `neighborhood_suggestion_claim`'s note asserts that it does: *"The level is a DECLARED TOKEN rather than a free placeholder, so this class discriminates."*

**This is the fifth turn of this issue's own defect** — a declaration written in a notation its reader cannot consume, with a note asserting the property it fails to have. The four before it: a literal `\n` in a form; the `lines:` array whose members are escaped whole; `neighborhood_empty`'s stale literals; and the near-bare gloss class that would have hidden a stale claim form.

**What is owed:** declare the token as an object with an anchored `shape`, verify against `tokenFragment` rather than by reading, and correct or remove the note's claim. The coverage case's per-line bareness test will not catch this one — the class is structurally non-bare — so the verification is a direct assertion that the compiled matcher **refuses** `  x [notalevel]`.

### 2. Field 2 names the batch instance, where the ruling's example names the settled member

The row renders:

```
- N1 — from the same Batch as the settled set (q_a/stub)
```

against #686 disposition 3's own example, *"from the same Batch as L88 (2026-08-13)"* — the settled Strand the candidate shares a Batch with, and the sitting's date. `q_a/stub` is the substrate's internal instance key, which is the token the code's own comment says the row exists to spare the reader.

The ruling's parenthetical is an **example rather than a form**, so this is a reading of it and not a breach. But naming the settled member requires the enumerator to record **which seed reached each suggestion**, which it does not today — `reached_by` carries `{substrate, instance}` and no seed. That is an enumerator change, which is why it lands here beside the Gloss rather than being improvised.

**Both items belong with this issue's own subject:** what the row's fields carry and where their values come from.

**Comment by tim-nish** [comment]
policy check: no remedy-shape conflict

Evaluated revision: title+body as of 2026-08-28T12:23:20Z (sha256 f68a255108ac).

Read as a **bounded-read decision plus three reconciliations**, not as an individual-prohibition shape: the issue's own `remedy:` declares `detects: … ; constrains: the row can no longer render an unquoted identifier in a field the grammar calls a served Gloss`, which is a constrain-shaped remedy over what the pipeline can produce, not a per-surface never-show rule.

**Owner disposition, 2026-08-28 — the fetch arm.** The report path makes the shard read, bounded by the rows the display cap already selected and by those rows' own tags. `gloss unrecorded` stays as the disclosed miss rather than becoming the resting decision, and §13.4's four-field promise stands unamended.

The served position the gate leaned on, quoted verbatim at its pin:

> "every enumerated class renders including its zero, and an empty class says so rather than being omitted" — the failure named is *silence*, and silence is indistinguishable from *not checked*.

`consulted: product-lab@b20d85ea9c2a6ba24542e7caa003ef42efce33b2 topics/archive/knowledge-architecture.md:57`
  request_id: 72840216-26ed-4988-bc77-d00acfc35d5a
  outcome: discriminating
  disposition: escalated
  query: Does an enumerated class owe rendering its own absence, and is a disclosed absence the fallback for a failed fetch rather than a licence to not fetch?
    axis: conduct
    facet: decision
    hit: none

The reading, stated as a reading: that line licenses the disclosure as the **fallback for a miss** and says nothing about declining to look. The bound itself is not novel — `resolveHeadlines` (kogaki#528) already fetches the union of the given members' own tags and already refused only the corpus-wide prefetch §9 names.

**Scope, classified spec.** Four items land: the bounded fetch and join; `neighborhood_suggestion_gloss` reconciled to the fetch arm; `NeighborhoodLevel` re-declared in a shape `tokenFragment` can consume, with its overstating note corrected; and field 2 naming the settled member the candidate shares a Batch with, which needs the enumerator to record which seed reached each suggestion.

**Comment by tim-nish** [comment]
## Carried from PR #693 round 1: the Gloss fetch addresses one shard namespace

#693 lands the bounded fetch this issue's gate ruled. One fork it does **not** settle, raised at round 1 and routed here:

`resolveHeadlines` addresses `lessons/<tag>` and only that — it was written for the Brief lane (kogaki#528), whose members are settled Lessons, so one namespace was the whole set there. **The neighborhood's are not.** `neighborhoodOf` indexes every served record carrying a `slug` and stamps `family` from its `kind`; `counts.by_family` exists to count the split; and `cmdView` — the surface this fetch borrows its bound from — reads **both** namespaces for exactly this reason.

So a `journey`-family suggestion judged `core` reaches no shard on any pull.

**#693 discloses it rather than widening the bound.** The row renders its own marker — `⟨no Gloss shard addressed — this row's family or tags reach none, and the report path reads the lessons namespace only⟩` — distinct from the read-and-carried-nothing marker, because rendering the second as the first asserts a read that never happened. That costs no reads and settles nothing.

**The fork itself is open:** should the report path address the `journeys/` namespace too? It roughly doubles the shard reads for a section that renders at most ten rows, and it is the same read-budget question the 2026-08-28 gate was raised for — the ruling reached the `lessons/` fetch and stopped there, so the implementing sitting did not extend it alone.

Note the bound is per-tag, not per-row: `cmdView` fetches both namespaces over the same tag union, so the cost is a second shard **per tag**, not per suggestion. That may make the answer cheap; it is still not the sitting's to give.

**Comment by tim-nish** [comment]
## The namespace fork's cost shape, corrected (PR #693 round 2)

The carry above described the `journeys/` read as if it were a new tag. It is not, and the difference matters for the fork:

`resolveHeadlines` is handed every shown row's tags **without filtering by family**, so a tagged journey row's tags already enter the shard union and a `lessons/<tag>` shard **is** read for them — that shard simply cannot carry a journey slug. So the read budget this fork governs is *already* being spent on rows the marker says nothing carries.

**The incremental cost of reading `journeys/` is therefore a second shard per tag already in the union, not a new tag** — the shape `cmdView` already pays (§9). That may make the answer cheap; it is still the owner's.

PR #694 (successor to #693) renders the disclosure precisely rather than loosely: `⟨no Gloss shard carries this row — its family is outside the lessons namespace this path reads, or it carries no tag⟩`.

**Comment by tim-nish** [comment]
The implementation is now carried by **#695**, not by a PR under this issue. #694 was complete and green but could not merge: it inherited #693's unadjudicated round-2 blocking finding through the supersession chain, and every further PR would inherit it too. #695 re-lands the work on an unjoined history and carries the chain defect itself.

This issue keeps exactly one open item: **whether the fetch should address the `journeys/` shard namespace**, with the corrected cost shape (a second shard per tag already in the union, not a new tag).

**Comment by tim-nish** [comment]
## Carrier arbitration — vitality trip, dispositioned `proceed this run`

The `/ship-cycle 689` sitting's vitality read **tripped** and the gate was raised as the run's first act on this carrier:

```
runs          6  (1 issue stamp + 5 PR-space touches: #692, #693, #694, #696, #697)
body          1 revision judged — no mutation across runs
descendant    PR #694 (closed)
TRIP — 6 runs against this carrier (trip at 3)
```

**Owner selection: proceed this run — answer the residual forks and close.** The trip **re-arms** rather than being silenced.

**The ground.** This carrier's implementation licence is discharged: the fetch arm was ruled 2026-08-28, landed via #695, and is exercised by five registered assertions. What it kept absorbing afterwards is residue, which is the umbrella shape the served record names —

> "**The filing-time rule: one issue = one closable act**, and a spec's evolution never shares a carrier with its implementation license."

> "an issue carrying an UNDECIDED FORK is a drift generator … **The remedy is to answer the fork in the filing**, citing the prior implementation as the authority, rather than to file the fork."

`consulted: product-lab@b20d85ea9c2a6ba24542e7caa003ef42efce33b2 topics/claude-code-ops.md:79,61`

Minting a successor for the remainder was the declined alternative, on those two lines plus `:78` — a successor for a non-gating residual finding owes stated reachability or an explicit owner promotion, because "termination requires at least one exit whose cost does not reproduce the process".

**The two residual forks, both answered at the same gate:**

1. **The namespace fork — `journeys/` is read too.** `resolveHeadlines` addressed `lessons/<tag>` only, so a `journey`-family suggestion reached no shard. The corrected cost shape from PR #693 round 2 stands: a tagged journey's tags are already in the shard union, so this is a **second shard per tag already in the union**, not a new tag — the shape `cmdView` already pays under §9. §9's bound is unbroken: still tag-scoped, still bounded by the rows that render, still no whole-corpus prefetch.

2. **The degraded-seam fourth state — distinguished now.** Where the seam itself is unreachable, no shard is read at all and every entry returns unfound, so a tagged lesson row rendered the read-and-carried-nothing marker — asserting a read that never happened. §13.4 recorded it as unrepaired and said the repair belonged with the read-budget fork; that fork is this one, so it lands here.

The carrier-read whole with its thread was performed per `policy/consultation-map.md` entry 3 before this disposition was written; the thread is what established that exactly one item remained open, which a body-only read would have got wrong in the other direction.

The implementing PR follows in this sitting and closes this issue.
